channel-sdk/src/main/java/com/zello/channel/sdk/SessionContextAndroid.kt

```diff
@@ -1,7 +1,15 @@
 package com.zello.channel.sdk
 
+import android.Manifest
 import android.content.Context
+import android.content.pm.PackageManager
 import android.os.Handler
+import android.support.v4.content.ContextCompat
```

Comment:
I believe this needs to be
```
import androidx.core.content.ContextCompat
```
Can't mix the two.

Reply:
This project hasn't been updated for androidx yet, so it shouldn't be using it at all outside of some test (JUnit) classes

Reply:
Ah... so we use androidx for test dependencies only at this point. Would you try to switch the rest to androidx though? It should be really straightforward.  It the gradle file it would mean replacing 
```
implementation 'com.android.support:appcompat-v7:27.1.1'
```
with
```
implementation "androidx.appcompat:appcompat:1.1.0"
```

Reply:
Sounds like I'll also need to update the Android SDK version to 28 or 29? It's currently at 27. Is there anything I should look out for when I do that?

Reply:
Not that I can think of right now. The latest SDK would be great. It's not like there's anything specific that we have in the SDK that will break simply due to upgrading the target API.